abrechnung/config.py

```diff
@@ -14,11 +14,11 @@
         "database": fields.Nested(
             Schema.from_dict(
                 {
-                    "host": fields.Str(),
+                    "host": fields.Str(required=False, missing=None),
```

Comment:
the missing key has been deprecated, I believe the correct option now is `load_default`

Reply:
will change that.